pkgs/development/libraries/gmp/5.1.x.nix

```diff
@@ -12,6 +12,8 @@ stdenv.mkDerivation (rec {
 
   nativeBuildInputs = [ m4 ];
 
+  patches = stdenv.lib.optional stdenv.isDarwin ./need-size-t.patch;
```

Comment:
This changes stdenv unfortunately. Should be `patches = if stdenv.isDarwin then [ ./need-size-t.patch ] else null` to avoid that.